src/main/java/frc/robot/subsystems/gripper/GripperIO.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.*;
+
+public interface GripperIO {
+
+    GripperInputs inputs = new GripperInputs();
+
+    void setSpeedMotorPower(double power);
```

Comment:
Maybe choose a better name for this motor, like roller motor.

Reply:
84fa821bbcf5ff97709d179680bd8507dd11cb08

Reply:
I meant for you to change it everywhere.

Reply:
84fa821bbcf5ff97709d179680bd8507dd11cb08

Reply:
6cdb54b410aaadc666886503a5a5ae5e509ef41c